docs/calibration.md

```diff
@@ -1,22 +1,12 @@
 Helmi is continuously calibrated to ensure functionality. The calibration process involves a series of experiments, aimed at fine-tuning the parameters, necessary for controlling and measuring the qubits. In addition to calibration, we run benchmarks to obtain the figures of merit reflecting the current state of the quantum computer.
 
-!!! note
-
-    Calibration and benchmarking experiments are interleaved with regular user jobs in Helmi's job queue, as detailed in the [Running on Helmi](running.md) section. The calibration might therefore slightly increase the waiting time in the queue of regular user jobs.
-
```

Comment:
I believe this is still relevant. Even if the calibration runs at 2am, users may still experience increased queue times if they run over night for example.